qemu/tests/rng_bat.py

```diff
@@ -46,7 +45,7 @@ def check_driver_status(session, check_cmd, driver_id):
         check_cmd = check_cmd.replace("DRIVER_ID", driver_id)
         status, output = session.cmd_status_output(check_cmd)
         if "disabled" in output:
-            raise exceptions.TestFail("Driver is disable")
+            test.fail("Driver is disable")
```

Comment:
Doesn't it need "test" parameters in function defination? 
such as:    def check_driver_status(session, check_cmd, driver_id, test)